forge-gui/res/cardsfolder/upcoming/chiss_goria_forge_tyrant.txt

```diff
@@ -0,0 +1,17 @@
+Name:Chiss-Goria, Forge Tyrant
+ManaCost:6 R R
+Types:Legendary Creature Dragon
+PT:5/4
+K:Affinity:Artifact
+K:Flying
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME attacks, exile the top five cards of your library. You may cast an artifact spell from among them this turn. If you do, it has affinity for artifacts.
+SVar:TrigExile:DB$ Dig | Defined$ You | DigNum$ 5 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STPlay | Triggers$ Play1 | RememberObjects$ Remembered | ForgetOnMoved$ Exile | SubAbility$ DBCleanup
+SVar:STPlay:Mode$ Continuous | EffectZone$ Command | AffectedZone$ Exile,Stack | AddKeyword$ Affinity:Artifact | Affected$ Card.IsRemembered+Artifact | MayPlay$ True | Description$ You may play one of those cards until end of turn.
```

Comment:
Imo the Affinity results only from this MayPlay Permission, meaning if you'd find another way to cast it should probably not apply

there are not many but you could use _Squee, the Immortal_

maybe the static can be split with `MayPlaySource`...?

Reply:
Hmm it seems like Forge handles this situation correctly 
![image](https://user-images.githubusercontent.com/67333662/214561589-4f42bf67-85eb-4334-baf9-725d054a73dd.png)
![image](https://user-images.githubusercontent.com/67333662/214561624-b019df69-5497-4144-aba4-baf7e108cbb7.png)

Reply:
Fixed